app/models/volunteer.rb

```diff
@@ -8,5 +8,7 @@ class Volunteer < ApplicationRecord
   has_many :target_groups, through: :preferred_target_groups
   has_many :skills_volunteers
   has_many :skills, through: :skills_volunteers
+  has_one :highest_education
+  has_many :formal_educations, through: :highest_education
```

Comment:
this should be a `has_one` too. http://guides.rubyonrails.org/association_basics.html#the-has-one-through-association